modules/navier_stokes/include/utils/CellCenteredMapFunctor.h

```diff
@@ -209,6 +211,33 @@ CellCenteredMapFunctor<T, Map>::evaluate(const ElemSideQpArg &, const StateArg &
   mooseError("not implemented");
 }
 
+template <typename T, typename Map>
+typename CellCenteredMapFunctor<T, Map>::ValueType
+CellCenteredMapFunctor<T, Map>::evaluate(const NodeArg & node_arg, const StateArg &) const
+{
+  const Node * const node = node_arg.node;
+
+  try
+  {
+    return libmesh_map_find(*this, node->id());
```

Comment:
so the cellCenteredMapFunctor has node-cached values?
seems odd? I feel like node value of a cellCentered functor would be some sort of average of cell-centered values?

if you want a map then it would be a NodeMapFunctor ?